build/scripts/GitHub/GitHubPullRequest.class.psm1

```diff
@@ -35,31 +35,52 @@ class GitHubPullRequest {
 
     <#
         Gets the linked issues IDs from the pull request description.
-        .param $issueSection
-            The section of the pull request description that contains the issue ID.
         .returns
             An array of linked issue IDs.
     #>
-    [int[]] GetLinkedIssueIDs($issueSection) {
+    [int[]] GetLinkedIssueIDs() {
         if(-not $this.PullRequest.body) {
             return @()
         }
 
-        $issueRegex = "$issueSection(\d+)" # e.g. "Fixes #1234"
-        $issueMatches = Select-String $issueRegex -InputObject $this.PullRequest.body -AllMatches
+        $workitemPattern = "(close|closes|closed|fix|fixes|fixed|resolve|resolves|resolved) #(\d+)" # e.g. "Fixes #1234"
```

Comment:
Consider adding (^|\s) at the beginning, to make sure you match the beginning of the word.

As is, it would match 'discloses #1234' and 'postfix #23'

Reply:
Fixed.